<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>502</width>
    <height>668</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle">
   <string>Control panel for Serach Engine</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QTabWidget" name="tabWidget">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>0</y>
      <width>481</width>
      <height>641</height>
     </rect>
    </property>
    <property name="currentIndex">
     <number>1</number>
    </property>
    <widget class="QWidget" name="configTab">
     <attribute name="title">
      <string>config.json</string>
     </attribute>
     <layout class="QGridLayout" name="gridLayout">
      <item row="0" column="0">
       <layout class="QVBoxLayout" name="verticalLayout_4">
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_4">
          <item>
           <widget class="QPushButton" name="newConfig">
            <property name="text">
             <string>New config</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="openConfig">
            <property name="text">
             <string>Open config</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="saveConfigButton">
            <property name="enabled">
             <bool>false</bool>
            </property>
            <property name="text">
             <string>Save config</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="loadListFilesButton">
            <property name="enabled">
             <bool>false</bool>
            </property>
            <property name="text">
             <string>Load list files</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <widget class="Line" name="line">
          <property name="orientation">
           <enum>Qt::Orientation::Horizontal</enum>
          </property>
         </widget>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_5">
          <item>
           <widget class="QLabel" name="label">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="text">
             <string>Name:</string>
            </property>
            <property name="scaledContents">
             <bool>false</bool>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLabel" name="name">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="font">
             <font>
              <bold>true</bold>
             </font>
            </property>
            <property name="text">
             <string/>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_6">
          <item>
           <widget class="QLabel" name="label_2">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="text">
             <string>Version:</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLabel" name="version">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="font">
             <font>
              <bold>true</bold>
             </font>
            </property>
            <property name="text">
             <string/>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_7">
          <item>
           <widget class="QLabel" name="label_3">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="layoutDirection">
             <enum>Qt::LayoutDirection::LeftToRight</enum>
            </property>
            <property name="text">
             <string>maxResponses:</string>
            </property>
            <property name="alignment">
             <set>Qt::AlignmentFlag::AlignLeading|Qt::AlignmentFlag::AlignLeft|Qt::AlignmentFlag::AlignVCenter</set>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLineEdit" name="maxResponses">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="font">
             <font>
              <bold>true</bold>
             </font>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <widget class="Line" name="line_2">
          <property name="orientation">
           <enum>Qt::Orientation::Horizontal</enum>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="label_4">
          <property name="text">
           <string>Resources:</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QListWidget" name="listForFiles">
          <property name="enabled">
           <bool>false</bool>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
    <widget class="QWidget" name="tab_2">
     <attribute name="title">
      <string>requests.json</string>
     </attribute>
    </widget>
    <widget class="QWidget" name="tab">
     <attribute name="title">
      <string>answers.json</string>
     </attribute>
    </widget>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>502</width>
     <height>21</height>
    </rect>
   </property>
   <property name="accessibleName">
    <string/>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>openConfig</sender>
   <signal>clicked()</signal>
   <receiver>MainWindow</receiver>
   <slot>openConfigFile()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>237</x>
     <y>56</y>
    </hint>
    <hint type="destinationlabel">
     <x>6</x>
     <y>291</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>saveConfigButton</sender>
   <signal>clicked()</signal>
   <receiver>MainWindow</receiver>
   <slot>saveConfigFile()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>352</x>
     <y>56</y>
    </hint>
    <hint type="destinationlabel">
     <x>218</x>
     <y>61</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>loadListFilesButton</sender>
   <signal>clicked()</signal>
   <receiver>MainWindow</receiver>
   <slot>changeListFiles()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>362</x>
     <y>40</y>
    </hint>
    <hint type="destinationlabel">
     <x>361</x>
     <y>52</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>newConfig</sender>
   <signal>clicked()</signal>
   <receiver>MainWindow</receiver>
   <slot>newConfigFile()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>71</x>
     <y>49</y>
    </hint>
    <hint type="destinationlabel">
     <x>8</x>
     <y>73</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>openConfigFile()</slot>
  <slot>saveConfigFile()</slot>
  <slot>changeListFiles()</slot>
  <slot>newConfigFile()</slot>
 </slots>
</ui>
